internal/scaffold/ansible/build_dockerfile.go

```diff
@@ -46,10 +46,10 @@ func (b *BuildDockerfile) GetInput() (input.Input, error) {
 
 const buildDockerfileAnsibleTmpl = `FROM quay.io/operator-framework/ansible-operator:[[.ImageTag]]
 
-COPY watches.yaml ${HOME}/watches.yaml
+COPY ${pwd}/watches.yaml ${HOME}/watches.yaml
```

Comment:
These `${pwd}`'s aren't necessary. Were you seeing errors when building the operator without them?

Reply:
good catch.